js/index.js

```diff
@@ -1 +1,112 @@
 // Your code goes here
+/*
+mouseover
+ keydown
+ wheel
+ drag / drop
+ load
+ focus
+ resize
+ scroll
+ select
+ dblclick
+ */
+
+const myH1 = document.querySelector('h1');
+console.log(myH1);
+
+myH1.addEventListener("mouseover", (event) => {
+	console.log("h1 mouseovered");
+	myH1.style.color = 'green';
+});
+
+myH1.addEventListener("mouseleave", (event) => {
+	console.log("h1 mouseleave");
+	myH1.style.color = 'black';
+	myH1.style.background = 'white';
+});
+
+myH1.addEventListener("mousedown", (event) => {
+	console.log("mousedown");
+	myH1.style.background = 'blue';
+});
+
+myH1.addEventListener("mouseup", (event) => {
+	console.log("mouseup");
+	myH1.style.background = 'white';
+});
+
+//unclear how to change background color just of selected text (character by character)
```

Comment:
I think you're practically looking for the [focus pseudo-selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus) in CSS. That's a good question, and I'm not entirely sure how to go about doing that with JS.